pkg/auth/providers/ldap/ldap_client.go

```diff
@@ -521,6 +549,5 @@ func (p *ldapProvider) RefetchGroupPrincipals(principalID string, secret string)
 	if err != nil {
```

Comment:
Since this will generate group principals using ldap's attributes, we'll need to change them according to the saml ones just like you're doing for searchPrincipals